CMakeLists.txt

```diff
@@ -271,3 +298,60 @@ foreach(LIB Luau.Ast Luau.Compiler Luau.Config Luau.Analysis Luau.CodeGen Luau.V
         endif()
     endif()
 endforeach()
+
+configure_package_config_file(
+    "${CMAKE_CURRENT_SOURCE_DIR}/LuauConfig.cmake.in"
+    "${CMAKE_CURRENT_BINARY_DIR}/LuauConfig.cmake"
+    INSTALL_DESTINATION "lib/cmake/Luau"
+)
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/LuauConfigVersion.cmake"
+    VERSION "${PROJECT_VERSION}"
+    COMPATIBILITY AnyNewerVersion
+)
+export(
+    TARGETS Luau.Ast Luau.Analysis Luau.CodeGen Luau.VM Luau.Compiler Luau.Config Luau.Common Luau.VM.Internals
```

Comment:
Luau.VM.Internals should not be included.

Reply:
I think the problem I had before when it wasn't there was this:
```
  export called with target "Luau.CodeGen" which requires target
  "Luau.VM.Internals" that is not in any export set.
```

I'm not sure what the best way to deal with this is yet.

Reply:
It looks like a bunch of headers, which would be exported with this, include VM internal headers.

Until that gets resolved Luau.VM.Internals needs  to be exported.